code/modules/research/designs/weapon_designs.dm

```diff
@@ -99,7 +112,7 @@
 	category = list(
 		RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_AMMO
 	)
-	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
+	departmental_flags = DEPARTMENT_BITFLAG_SECURITY | DEPARTMENT_BITFLAG_SERVICE
```

Comment:
this seems excessive to give service dart shells, they are one of the strongest shells in the gmae